unlock-app/src/unlockTypes.ts

```diff
@@ -69,3 +69,11 @@ export interface Action {
 
 // TODO: Use this where we have TS files
 export type Dispatch = (action: Action) => any
+
+// This is currrently the way ethers checks the keystore format. TODO:
```

Comment:
See: https://github.com/ethers-io/ethers.js/blob/master/src.ts/utils/json-wallet.ts#L17

We can make this type more concrete, but I'm not certain how solid the other properties of the keystore object are. At any rate, this fulfills what it's meant to, because it should prevent anyone from accidentally using a decrypted key as an encrypted one.